README.md

```diff
@@ -1 +1,30 @@
-# springtrader-marketSummary
\ No newline at end of file
+# springtrader-marketSummary
```

Comment:
Could you lowercase the s in marketsummary?

Reply:
Changed it!